ratbag/drivers/sinowealth.py

```diff
@@ -166,10 +166,7 @@ def run(self, rodent: ratbag.driver.Rodent) -> Reply:
             )
 
         # Parse to the properties
-        rspec = [
-            Spec("B", "report_id"),
-            Spec("B", "cmd"),
-        ] + self.reply_spec
+        rspec = self.reply_spec
```

Comment:
this seems strange - in the libratbag driver the first two fields of `struct sinowealth_config_report` are the `report_id` and `command_id`. So I'd expect these to be in place here? If you drop them from the `Spec` list your parsing will be off by 2 bytes now.

Reply:
They are already in the config report struct.
Before this change the spec was like this:
- Report ID
- CMD
- Report ID
- CMD
- Actual config report data..

Reply:
oh, woops, sorry about the noise. This approach with a standard header is more interesting for drivers with lots of reports (hidpp20 for example) but here we have only 2 I think, so let's skip it.

You won't need `rspec` then btw, you can just pass `self.reply_spec` to the parser. but that's a nitpick only.